src/transports/sentry_transport_winhttp.c

```diff
@@ -71,7 +105,12 @@ sentry__winhttp_transport_start(
     // ensure the proxy starts with `http://`, otherwise ignore it
     if (opts->proxy && strstr(opts->proxy, "http://") == opts->proxy) {
         const char *ptr = opts->proxy + 7;
```

Comment:
Apparently our Windows transport doesn't take https:// proxies. This should be documented, since we now also read the `HTTPS_PROXY` environment variable, which on this platform would get ignored.